modules/local/metaspades.nf

```diff
@@ -0,0 +1,49 @@
+// similar to nf-core/mag spades local module: https://github.com/nf-core/mag/blob/master/modules/local/spades.nf except with newer version of spades and only emits gzipped versions of assemblies
+
+process METASPADES {
+    tag "$meta.id"
+
+    conda (params.enable_conda ? "bioconda::spades=3.15.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/spades:3.15.3--h95f258a_0' :
```

Comment:
shouldn't we use metaspades instead of spades? or am i missing something (I almost always use megahit so forgive me if i'm being naive!)

Reply:
are they included in the same package or something?

Reply:
they are included in the same spades package, you just call the `metaspades.py` script from spades